libc/cmake/modules/LLVMLibCTestRules.cmake

```diff
@@ -656,13 +649,12 @@ function(add_libc_hermetic test_name)
 
   if(LIBC_TARGET_ARCHITECTURE_IS_AMDGPU)
     target_link_options(${fq_build_target_name} PRIVATE
-      ${LIBC_COMPILE_OPTIONS_DEFAULT}
-      -mcpu=${LIBC_GPU_TARGET_ARCHITECTURE} -flto -Wno-multi-gpu
+      ${LIBC_COMPILE_OPTIONS_DEFAULT} -Wno-multi-gpu
+      -mcpu=${LIBC_GPU_TARGET_ARCHITECTURE} -flto
+      "-Wl,-asdfasdfasdf"
```

Comment:
Is `asdfasdfasdf` something that's specific to LLVM or AMDGPU? If so, what's the story behind the name?

Reply:
Whoops.

Reply:
The tests pass, so I wonder why this doesn't break anything.

Reply:
Huh, interesting

Reply:
That made it into the LLVM 19 fork, how embarrassing.